dotcom-rendering/src/components/ManyNewsletterSignUp.importable.tsx

```diff
@@ -6,18 +6,29 @@ import {
 	space,
 } from '@guardian/source-foundations';
 import { Button, SvgCross } from '@guardian/source-react-components';
-import type { ChangeEventHandler } from 'react';
-import { useCallback, useEffect, useState } from 'react';
+import type { ChangeEventHandler, ReactEventHandler } from 'react';
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { ReCAPTCHA } from 'react-google-recaptcha';
+import {
+	getCaptchaSiteKey,
+	mockRequestMultipleSignUps,
+	reportTrackingEvent,
+	requestMultipleSignUps,
+} from '../lib/newsletter-sign-up-requests';
 import { Flex } from './Flex';
 import { ManyNewslettersForm } from './ManyNewslettersForm';
 import { BUTTON_ROLE, BUTTON_SELECTED_CLASS } from './NewsletterCard';
 import { Section } from './Section';
 
 interface Props {
-	/** The endpoint to send the sign-up request to. An empty string will
-	 * make the component use test results instead of making an actual request.
+	/**
+	 * Whether to use the mocked request function.
+	 *
+	 * This option is to allow a functional Story to be published for the component.
+	 * TO DO - find a better way to do that!
 	 */
-	apiEndpoint: string;
+	useMockedRequestFunction?: boolean;
```

Comment:
There’s a global mock for requests if helpful:

https://github.com/guardian/dotcom-rendering/blob/beb8a4899d796641761b140659c837c9505f91bb/dotcom-rendering/src/lib/mockRESTCalls.ts#L130

Reply:
Thanks - I hadn't seen that. I've pushed a change to use the mock in the story instead of having that prop.